<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
border:none

}
QPushButton:hover
{
	background-color: rgb(255, 45, 17);
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>290</width>
     <height>50</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>9</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/MusicPlay/images/menu.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>9</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/MusicPlay/images/starting.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>9</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/MusicPlay/images/start.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>9</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/MusicPlay/images/last.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>9</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/MusicPlay/images/next.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>9</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/MusicPlay/images/pause.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>251</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="playCourseSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>20</width>
     <height>91</height>
    </rect>
   </property>
   <property name="value">
    <number>30</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListView" name="Lyrics">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblLyrics1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>未检查到歌词</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblLyrics2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请检查歌词路径</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>281</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="curLable">
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="totalLable">
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
